src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp

```diff
@@ -99,9 +99,28 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
         return make_unique<ImplType>(best_kernel);
     }
 
+    void build_kernels(kernels_cache& kernels_cache) override {
+        auto kernels = kernels_cache.compile(get_kernels_source());
+        set_kernels(kernels);
+    }
+
```

Comment:
Should we have this method in impl?
We can just compile outside, and set_kernels to this impl ?

Reply:
rollback it.